engine/src/main/java/org/terasology/rendering/dag/nodes/AmbientOcclusionPassesNode.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.terasology.rendering.dag;
```

Comment:
I'm thinking, if everything is going into a "nodes" package it might be appropriate to remove the word "Node" from the name of every node class. 

By the way, this reminds me of a trick to keep PRs "cleaner". When you do a trivial but necessary change, like moving a bunch of classes into a sub-package, wrap that into its own commit and make sure the reviewers know that commit is trivial. That way the reviewers know to focus on the following commit(s) and have far fewer changes to go through.

Look for example at my #1745 PR: each commit has a specific purpose, the idea being reviewers have the chance to focus on commits that are doing serious changes rather than simple after-change maintenance.